__integrations__/models/profile.test.ts

```diff
@@ -478,4 +479,41 @@ describe("Models |> Profile", () => {
       context.dispose();
     }
   );
+
+  it.each`
+    inputValue     | expectedValue
+    ${undefined}   | ${"UNSET"}
```

Comment:
ditto

Reply:
done in https://github.com/pagopa/io-functions-commons/pull/297/commits/25f802f12c34ec047393044f870ae527936bc452